.github/workflows/win.yml

```
@@ -0,0 +1,44 @@
+name: windows
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        #CONDA_PY: ["36", "37"]
+        CONDA_PY: ["37"]
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-activate-base: true
```

I believe you should set it to false, as you are pining conda to ``4.8.3``
```suggestion
        auto-activate-base: false
```

---

and o top of that I believe would be better to use the ``miniconda-version`` and set it to ``4.8.3``

---

```suggestion
        auto-activate-base: false
        miniconda-version: "4.8.3"
```

---

Yes, I agree. Thanks for your suggestions!

---

I forgot to mention to also specify ``conda-version`` to guarantee the ``4.8.3`` will be used

---

Done. Thanks!